Frontend/src/Components/PPC_MPPT/PPC_MPPT.js

```diff
@@ -34,43 +37,135 @@ export default function PPC_MPPT(props) {
-                    max={CONSTANTS.string1_temp.MAX}
-                />
-                <RangeCell
-                    fontSize='1vw'
-                    label='Cell String 2 Temp'
-                    data={props.data?.string2_temp[0] ?? -1.0}
-                    digits={3}
-                    unit="&#8451;"
-                    min={CONSTANTS.string2_temp.MIN}
-                    max={CONSTANTS.string2_temp.MAX}
-                />
-                <RangeCell
-                    fontSize='1vw'
-                    label='Cell String 2 Temp'
-                    data={props.data?.string3_temp[0] ?? -1.0}
-                    digits={3}
-                    unit="&#8451;"
-                    min={CONSTANTS.string3_temp.MIN}
-                    max={CONSTANTS.string3_temp.MAX}
-                />
+                <Grid
+                    py='1rem'
+                    templateColumns="repeat(4, 1fr)"
+                    templateRows="repeat(4, 1fr)"
+                    gap={0.5}
+                >
+                    <Spacer></Spacer>
+                    <GridItem
+                        bg='gray'
+                        color='black'
+                        fontWeight='bold'
+                        height='20px'
+                        fontSize={10}
+                        alignItems='center'
+                        display='flex'>Temp(ºC)</GridItem>
+                    <GridItem
+                        bg='gray'
+                        color='black'
+                        fontWeight='bold'
+                        height='20px'
+                        fontSize={10}
+                        alignItems='center'
+                        display='flex'>Voltage(V)</GridItem>
+                    <GridItem
+                        bg='gray'
+                        color='black'
+                        fontWeight='bold'
+                        height='20px'
+                        fontSize={10}
+                        alignItems='center'
+                        display='flex'>Current(A)</GridItem>
+                    <GridItem
+                        alignItems='right'
+                        fontSize={12}>String 1</GridItem>
+                    <GridItem>
+                        <RangeCell
+                            fontSize='1vw'
+                            data={props.data?.string1_temp[0] ?? -1.0}
+                            digits={3}
+                            min={CONSTANTS.string1_temp.MIN}
+                            max={CONSTANTS.string1_temp.MAX}
+                        />
+                    </GridItem>
+                    <GridItem>
+                        <RangeCell
+                            fontSize='1vw'
+                            data={props.data?.string1_V_in[0] ?? -1.0}
+                            digits={3}
+                            min={CONSTANTS.string1_V_in.MIN}
+                            max={CONSTANTS.string1_V_in.MAX}
+                        />
```

Comment:
Since there's no label or units, instead of `RangeCell`s, let's use `Text`s and `RangeBar`s within `VStack`s or `Flex`s (probably `VStack`s), with the items being center aligned. This will allow the data readouts to be centered within the `GridItem`, whereas in the `RangeCell`, there is currently a space to the left of the readouts that is throwing them off-center.